test/plots/scatterPlotTests.ts

```diff
@@ -2,395 +2,386 @@
-      let plot = new Plottable.Plots.Scatter()
-                                   .x((d: any) => d.x, xScale)
-                                   .y((d: any) => d.y, yScale)
-                                   .addDataset(dataset)
-                                   .addDataset(dataset2);
-      plot.renderTo(svg);
-
-      let points = d3.selectAll(".scatter-plot path");
-      let d0 = dataset.data()[0];
-      let d0Px = {
-        x: xScale.scale(d0.x),
-        y: yScale.scale(d0.y)
-      };
-
-      let expected: Plottable.Plots.PlotEntity = {
-        datum: d0,
-        index: 0,
-        dataset: dataset,
-        position: d0Px,
-        selection: d3.selectAll([points[0][0]]),
-        component: plot
-      };
-
-      let closest = plot.entityNearest({ x: d0Px.x + 1, y: d0Px.y + 1 });
-      TestMethods.assertPlotEntitiesEqual(closest, expected, "it selects the closest data point");
-
-      yScale.domain([0, 1.9]);
-
-      let d1 = dataset.data()[1];
-      let d1Px = {
-        x: xScale.scale(d1.x),
-        y: yScale.scale(d1.y)
-      };
-
-      expected = {
-        datum: d1,
-        index: 1,
-        dataset: dataset,
-        position: d1Px,
-        selection: d3.selectAll([points[0][1]]),
-        component: plot
-      };
-
-      closest = plot.entityNearest({ x: d1Px.x, y: 0 });
-      TestMethods.assertPlotEntitiesEqual(closest, expected, "it ignores off-plot data points");
-
-      svg.remove();
+      it("is initialized correctly", () => {
+        let data = [{ x: 0, y: 0 }];
+        let dataset = new Plottable.Dataset(data);
+        plot.addDataset(dataset);
+
+        assert.isDefined(plot.attr("fill"), "default color is defined");
+        assert.isDefined(plot.animator(Plottable.Plots.Animator.MAIN), "main animator is defined");
+        assert.isDefined(plot.size(), "size() is defined");
+        assert.isDefined(plot.symbol(), "symbol() is defined");
+
+        svg.remove();
+      });
     });
 
-    it("can retrieve entities in a certain range", () => {
-      let svg = TestMethods.generateSVG(400, 400);
-      let xScale = new Plottable.Scales.Linear();
-      let yScale = new Plottable.Scales.Linear();
+    describe("the Accessors properly access data, index and Dataset", () => {
+      const SVG_WIDTH = 400;
+      const SVG_HEIGHT = 400;
+      let svg: d3.Selection<void>;
+      let plot: Plottable.Plots.Scatter<number, number>;
+      let dataset: Plottable.Dataset;
+      let data: any[];
 
-      let dataset = new Plottable.Dataset([{x: 0, y: 0}, {x: 1, y: 1}]);
-      let dataset2 = new Plottable.Dataset([{x: 1, y: 2}, {x: 3, y: 4}]);
-      let plot = new Plottable.Plots.Scatter();
-      plot.x((d: any) => d.x, xScale)
-          .y((d: any) => d.y, yScale)
-          .addDataset(dataset)
-          .addDataset(dataset2);
-      plot.renderTo(svg);
+      beforeEach(() => {
+        svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
```

Comment:
Is there a reason why we need a specific SVG_WIDTH and height? If not, let's use the default method.